demodoctrine/js/.webpack/common.js

```diff
@@ -0,0 +1,83 @@
+const fs = require('fs');
+const path = require('path');
+const webpack = require('webpack');
+const CleanWebpackPlugin = require('clean-webpack-plugin');
+const ExtractTextPlugin = require('extract-text-webpack-plugin');
+const cssExtractedFileName = 'theme';
+
+// PrestShop folders, we use process.env.PWD instead of __dirname in case the module is symlinked
+const psRootDir = path.resolve(process.env.PWD, '../../../');
+const psJsDir = path.resolve(psRootDir, 'admin-dev/themes/new-theme/js');
+const psComponentsDir = path.resolve(psJsDir, 'components');
+
+module.exports = {
+  externals: {
+    jquery: 'jQuery',
+  },
+  entry: {
+    quotes: './quotes',
+    quote_form: './quotes/form',
+  },
+  output: {
+    path: path.resolve(__dirname, '../../views/js'),
```

Comment:
This path looks strange for assets, maybe a public folder ? Or I'm missing something maybe ^^

Reply:
This is the generic path we used since years for public assets in many modules :sweat_smile:

Reply:
Yeah historically the modules assets are supposed to be in the `views` folder.. So I follow this convention and build the assets in this folder

Reply:
Damn... that's.... uh.

We don't change to avoid breaking changes?

Reply:
It's not exactly a breaking change, it's more a convention The idea is that a module developer is used to find assets at this place

We might (should?) change it one day but we would need to define the new expected structure, document it and apply it in the native modules